### PR TITLE
Add ADR for Github integration

### PR DESCRIPTION
**Pull request by brenetic**
This is a retrospective ADR regarding the reasons for the integration of
Github into the Form Bulder platform.

### CODE REVIEW

**Comment by brenetic** [comment]
@tomas-stefano @mattempty @sequencemedialimited Have attempted to gather the reasons for the integration with Github. Please do comment and add anything I might have missed.

File: decisions/0005-integration-with-github.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The decision to use Github, and git itself, as the versioning system for the forms was not recorded at the time so this is in place of that.
+
+The Form Builder platform needs a place to store the JSON metadata that the editor generates. This metadata is used by the Runner Node's in order to create the forms upon deployment.
+
+## Decision
+
+The main reasons for choosing git and Github:
+
+- Transparancy. All the forms are published in the open for the public to see.
```
Comment (tomas-stefano) [inline decisions/0005-integration-with-github.md]: Transparency

**Comment by mattempty** [comment]
Was collaborating one of the reasons or is that assumed in the versioning?

**Comment by brenetic** [comment]
@tomas-stefano @mattempty thanks for the comments. Have updated

**Pull request merged**